editor/editor_themes.cpp

```diff
@@ -86,17 +86,17 @@ static Ref<StyleBoxFlat> change_border_color(Ref<StyleBoxFlat> p_style, Color p_
 	return style;
 }
 
-Ref<ImageTexture> editor_generate_icon(int p_index, bool dark_theme = true, Dictionary *p_colors = NULL) {
+Ref<ImageTexture> editor_generate_icon(int p_index, bool dark_theme = true, Dictionary *p_colors = NULL, float p_scale = EDSCALE, bool force_filer = false) {
```

Comment:
Typo, should be `force_filter` no?

Reply:
Actually it should even be `p_force_filter` to follow the convention (and `p_dark_theme` a couple parameters before).